packages/jest-runner/src/index.js

```diff
@@ -108,14 +108,16 @@ class TestRunner {
         if (watcher.isInterrupted()) {
           return Promise.reject();
         }
+
         await onStart(test);
-        return worker({
+
+        return worker.worker({
           config: test.context.config,
           globalConfig: this._globalConfig,
           path: test.path,
           rawModuleMap: watcher.isWatchMode()
             ? test.context.moduleMap.getRawModuleMap()
-            : null,
+            : undefined,
```

Comment:
why?

Reply:
Mistake I guess. I reverted that one.

Reply:
Oh, I found why. The expressed type was incorrect. I fixed it in [a second commit](https://github.com/facebook/jest/pull/4825/commits/b5674abcf9dac5c76cf7909f9b2035d3c9166aa7).